tailwind.config.js

```diff
@@ -15,10 +15,13 @@ module.exports = {
       grayLight: "#777777",
       grayLightest: "#f9f7f6",
       white: "#ffffff",
-      grayLightest: "#f9f7f6"
+      greenLight: "#D4EDDA",
+      greenDark: "#155724"
     },
     screens: {
```

Comment:
I think you might need to put the additional screen breakpoints in an "extend" object, otherwise the d[efault ones get replaced as well ](https://tailwindcss.com/docs/screens#overriding-the-defaults)(which would break the contact page), as in:
```
extend: {
      screens: {
        tablet: "768px",
        desktop: "992px",
        xl: "1200px"
      }
    }
```